README.md

```diff
@@ -33,15 +55,150 @@ items:
     tolerations: []
 ```
 
-### Signed Certificate
+#### Signed Certificate:
 A signed certificate ensures that there not unnecessary issues when performing the training pipeline.
 
 To deploy a signed certificate in cluster follow [trusted cluster cert](signed-certificate/README.md)
 
-### Object Storage
-This solution requires object storage to be in place either through S3 or using Noobaa.
+#### Deploy Teacher and Judge Models:
+
+In addition to model training, InstructLab also performs Synthetic Data Generation (SDG) and Model Evaluation. In both cases another LLM is required to complete these steps. Since these models do not change frequently, we recommend serving them independently of the specific InstructLab pipeline so that they can be used as a shared resources across the organization.
+
+1. Deploy the Teacher Model following these [instructions](/kubernetes_yaml/mixtral_serve/README.md).
+2. Deploy the Judge Model following these [instructions](/kubernetes_yaml/prometheus_serve/README.md).
+
+Once these two model servers are deployed, we need to add the following configmaps and secrets to our namespace so that the InstructLab pipeline can successfully communicate with each model.
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: teacher-server
+  namespace: <YOUR_NAMESPACE>
```

Comment:
You don't need to specify the namespace every time.. if you submit a resource without `.metadata.namespace` it gets created in the currently selected namespace... so.. we can just omit it here and de-cluter.